app/Http/Controllers/Backend/Transport/TrainCheckinController.php

```diff
@@ -136,10 +136,10 @@ private static function createTrainCheckin(
 
         //Note: Compare with ->format because of timezone differences!
         $firstStop = $trip->stopovers->where('train_station_id', $origin->id)
-                                     ->where('departure_planned', $departure->format('Y-m-d H:i:s'))
+                                     ->where('departure_planned', $departure)
```

Comment:
Is this really working? I had issues with this in the past in this function.

Reply:
It's not not working. Laravel will always cast datetime-castables to `Y-m-d H:i:s` inside its query builder. If you're supplying a datetimestring, the query builder will parse it to carbon and then render it to `Y-m-d H:i:s`.

On top of that, `departure_planned` has been changed to UTCDateTime, which needs an instance of carbon or a datetime string with a timezone, otherwise it will fail with "THIS IS A WONKY LOCAL TIME!!1" :D

You'll still need `->format('Y-m-d H:i:s')` in a `whereRAW`-condition but as long as it's passing through the query builder, we should supply the Carbon model. (This also saves RAM!!1 :D)